src/org/aavso/tools/vstar/ui/model/list/ArbitraryFormatRawDataColumnInfoSource.java

```diff
@@ -69,7 +71,9 @@ public int getColumnCount() {
 
 	@Override
 	public Collection<String> getColumnNames() {
-		return COLUMN_NAMES.keySet();
+		List<String> names = new ArrayList<String>(COLUMN_NAMES.keySet());
+		names.addAll(ValidObservation.getDetailTitles().keySet());
```

Comment:
I don't know the codes well so this might not be applicable.
Logically, the dropdown in Observations tab > Regular Expression search is similar to the VeLa filter dropdown here. 

But it seems to be based on different logic: I did a search on `ValidObservation.getDetailTitles()` and couldn't find it elsewhere. 

If they are indeed based on different logic, consider to consolidate them.

Reply:
I'm occupied with a few things but will return to this in a day or so.

Reply:
@orionlee, thanks for the comment. The change led to the VeLa filter and the observation list regex search using the same approach, and not `ValidObservation.getDetailTitles()` (on the branch).